File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/WarehouseStateEventId.java
package org.dddml.wms.domain;


public class WarehouseStateEventId
{
    private String warehouseId;

    public String getWarehouseId()
    {
        return this.warehouseId;
    }

    public void setWarehouseId(String warehouseId)
    {
        this.warehouseId = warehouseId;
    }

    private Long version;

    public Long getVersion()
    {
        return this.version;
    }

    public void setVersion(Long version)
    {
        this.version = version;
    }

    WarehouseStateEventId ()
    {
    }

    public WarehouseStateEventId (String warehouseId, Long version)
    {
        this.warehouseId = warehouseId;
        this.version = version;
    }

}

File: Dddml.Wms.AdminUI/ts/WarehouseStateEventId.ts
namespace Dddml.Wms.Domain {

	export class WarehouseStateEventId {

		private warehouseId: string;

		getWarehouseId(): string {
			return this.warehouseId;
		}

		setWarehouseId(warehouseId: string) {
			this.warehouseId = warehouseId;
		}

		private version: string;

		getVersion(): string {
			return this.version;
		}

		setVersion(version: string) {
			this.version = version;
		}

	}
}

